content/en/docs/concepts/event-sources/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Event Sources
+description: Leverage multiple Event Sources to increase the power of Falco
+linktitle: Event Sources
+weight: 10
+aliases:
+- ../event-sources
+- ../event-sources/enable-event-sources
+---
+## Overview
+
+Falco evaluates streams of events against security rules to detect abnormal behavior. Events are consumed through various event sources, which define the origin, nature, and format of the streamed events.
+
+Falco natively supports the `syscall` event source, which is enabled by default, consuming events from the Linux Kernel via drivers. The [plugin system](/docs/event-sources/plugins/) allows Falco to extend its capabilities with new event sources.
+
+### How It Works
+
+Each event source operates in an isolated thread, where events are generated, consumed, and evaluated by Falco’s rule engine. Rules are partitioned by event source, meaning each rule applies to a specific source and is triggered exclusively by events from that source. Falco does not support correlating events from different sources.
+
+By default, Falco enables event collection from all known sources, including `syscall` and other data sources added through plugins. The behavior can be customized using Falco's configuration.
+
+## Configuring Event Sources
+
+Before enabling or disabling specific event sources, you must configure the Falco engine and/or plugins as required by your use case:
+
+- **Engine:** For [drivers](/docs/concepts/event-sources/kernel/), [gVisor](/docs/concepts/event-sources/gvisor/) or capture files. 
+- **Plugins:** Ensure plugins are loaded properly by following the [plugin usage guide](https://falco.org/docs/plugins/usage/#loading-plugins-in-falco).
+
+{{% alert title="Capture Files" color="primary" %}}
+
+When processing capture files, Falco can handle events from multiple sources without additional configuration.
```

Comment:
Is this true? Don't we need the appropriate plugin installed?

Reply:
It is somehow true since the source capability of the plugin is not required while requiring the plugin. However, the extraction capability will be necessary to extract fields. I will improve the text.

Reply:
alert removed for now